core/tests/test_apps.eos.conversions.py

```diff
@@ -0,0 +1,56 @@
+from common import *
+
+from apps.eos import helpers
+from trezor.messages.EosAsset import EosAsset
+
+class TestEosConversions(unittest.TestCase):
+    def test_eos_name_to_string(self):
+        names_in = [
+            10639447606881920736,
+            614251623682315968,
+            614251535012020768,
+            7754926748989239168,
+            14895601873759291472,
+            595056260442243600,
+        ]
+        names_out = [
+            'miniminimini',
+            '12345abcdefg', 
+            '123451234512',
+            'hijklmnopqrs',
+            'tuvwxyz12345',
+            '111111111111',
+        ]
+        for i, o in zip(names_in, names_out):
+            self.assertEqual(helpers.eos_name_to_string(i), o)
+
+    def test_eos_asset_to_string(self):
```

Comment:
please add a test case for an amount < 1 EOS

Reply:
Done